scripts/zenodo-bibtex.py

```diff
@@ -77,7 +77,8 @@ def get_bibtex(ids):
     assert firedrake_tag[18] == "."
     digits = map(str, range(10))
     assert all([a in digits for a in (firedrake_tag[10:18] + firedrake_tag[19:])])
-except:
+except:                    # noqa: E722
+    # We really want to swallow everything here
```

Comment:
if you really want to swallow everything, you could just write `except Exception:` I think